sigs.yaml

```diff
@@ -2717,7 +2717,15 @@ workinggroups:
     - github: zacharysarah
       name: Zach Corleissen
       company: Linux Foundation
-  meetings: []
+  meetings:
+  - description: Regular WG Meeting
+    day: Monday
+    time: "10:30"
+    tz: PT (Pacific Time)
+    frequency: monthly - second Monday of month
+    url: https://zoom.us/j/91522666403?pwd=R0FpT3pCZm84c2ZYT2xRTjI5emc1UT09
+    archive_url: https://bit.ly/k8s-wg-naming-agenda
+    recordings_url: TBD
```

Comment:
```suggestion
    recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP3BrAtDHyr8KTUfhBhCG7CD
```

Reply:
Since things involve generation here, I'm amending the commit rather than using the GH UI to accept this.  Fixed commit coming in just a moment..